pkg/rancher-desktop/main/commandServer/settingsValidator.ts

```diff
@@ -524,17 +524,7 @@ export default class SettingsValidator {
       return false;
     }
 
-    if (desiredValue !== currentValue) {
-      if (desiredValue === PathManagementStrategy.NotSet) {
-        errors.push(`${ fqname }: "${ desiredValue }" is not a valid strategy`);
-
-        return false;
-      }
-
-      return true;
-    }
-
-    return false;
+    return desiredValue !== currentValue;
```

Comment:
Given the simplification, we no longer need a specific `checkPathManagementStrategy` and can use `checkEnum` instead.